hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/server/JsonUtils.java

```diff
@@ -81,6 +83,22 @@ public static JsonNode readTree(String content) throws IOException {
     return MAPPER.readTree(content);
   }
 
+  public static HashMap<String, Object> getResponseMap(String response)
+      throws IOException {
+    return MAPPER.readValue(response,
+        new TypeReference<HashMap<String, Object>>() {
+        });
+  }
+
+  public static List<HashMap<String, Object>> readTreeAsListOfMaps(String json)
+      throws IOException {
+    ObjectMapper objectMapper = new ObjectMapper();
+    return objectMapper.readValue(json,
+        new TypeReference<List<HashMap<String, Object>>>() {
+        });
+  }
```

Comment:
The name `getResponseMap` is specific to the usage in namespace CLI.  Also, why do we need both methods?  Can we keep only `readTreeAsListOfMaps` and let it use `MAPPER`?